client/rest-high-level/src/main/java/org/elasticsearch/client/security/user/privileges/GlobalOperationPrivilege.java

```diff
@@ -27,65 +27,75 @@
 import java.util.Objects;
 
 /**
- * Represents generic global application privileges that can be scoped for each
- * application. The privilege definition, as well as the scope identifier, are
- * outside of the Elasticsearch jurisdiction.
+ * Represents generic global cluster privileges that can be scoped by categories
+ * and then by operations. The privilege definition, as well as the operation
+ * identifier, are outside of the Elasticsearch jurisdiction. Categories are
```

Comment:
I don't quite follow (or perhaps don't agree) with your point about the definition being "outside of the Elasticsearch jurisdiction". Elasticsearch definitely controls & evaluates these.